modules/gdscript/gdscript_parser.cpp

```diff
@@ -7991,6 +7991,8 @@ void GDScriptParser::_check_function_types(FunctionNode *p_function) {
 		if (_get_function_signature(base_type, p_function->name, return_type, arg_types, default_arg_count, _static, vararg)) {
 			bool valid = _static == p_function->_static;
 			valid = valid && return_type == p_function->return_type;
+			if (!p_function->return_type.has_type)
+				p_function->return_type = return_type;
```

Comment:
This should now use braces around the single-line block.

Reply:
done!